packages/block-library/src/details/edit.js

```diff
@@ -48,6 +48,7 @@ function DetailsEdit( { attributes, setAttributes, clientId } ) {
 			<InspectorControls>
 				<PanelBody title={ __( 'Settings' ) }>
 					<ToggleControl
+						__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the editor, add a Details block and see the block inspector.

<img width="273" alt="Details block inspector" src="https://github.com/user-attachments/assets/f28abb01-0c6f-4d63-87be-a5b2932caa66">